dailystats.py

```diff
@@ -26,6 +27,40 @@
 PEAK_DATE_OBJECT = datetime.strptime(PEAK_DATE_STRING, "%Y-%m-%d")
 
 
+def main():

```

Comment:
I've broken your module level code into function here, with `main` going first. The logic hasn't changed much, but to refactor it I've removed most of the "side effects" (e.g. data being initialised and then mutated)